localbot/AnonModule.cs

```diff
@@ -98,24 +143,40 @@ public async Task Blacklist(int num)
             {
                 if(u.AliasAs(num))
                 {
-                    AnonUser suspect = u;
-                    activeUsers.Remove(u);
-                    blacklist.Add(u);
-                    await ReplyAsync($"user was blacklisted");
-                } else
+                    u.blacklisted = true;
```

Comment:
```suggestion
                    activeUsers.Find(user => user.AliasAs(num)).blacklisted = true;
```

Pretty sure you can use some fancy C# predicate here if you want.

Not super sure it will be any more efficient tho. It's just cool, and maybe(?) a bit more readable?